_episodes/09-practical-examples.md

```diff
@@ -7,5 +7,86 @@ questions:
 objectives:
   - Use docker in combination with real research software
 keypoints:
--  
+  - Docker can be a solution to include the legacy script into your workflow with minimum effort.
+  - You can use intermediate files to let different Docker containers talk to each other.
 ---
+
+You are a social science researcher and are trying to establish how many people in a list are below the allowed drinking age.
+You are provided with a list of names and the corresponding birthdates.
+
+A former PhD student already did the first part and wrote a program to convert birthdates into the current age of the people. Unfortunately it was written in Python2.7 and there is no way to upgrade it to Python3 (technically there is, but psssst).
+
+As you are on of the best and brightest you have written a code to check the current age of people against the legal drinking age and output all people who are below it. You wrote it in Python 3.6. Your professor/boss/overlord and the rest of the group use python 3.3 in the work and cannot use your critical piece of software.
+
+Create two docker containers and provide your colleagues with a working docker workflow before they get angry!
+
+## Preparation work
+
+Before you start with a solution, you need to download the files you need:
+
+- the old script [convertbirthdaytoage.py](/code/convertbirthdaytoage.py) written by the former PhD in Python 2.7;
+- the new script [heckdrinkinglimit.py](/code/checkdrinkinglimit.py) you wrote in Python 3.6;
+- the [birthday data](/data/data_birthday.csv) required for this study.
+
+To simplify our workflow, let's put them all in the same working directory:
```

Comment:
Is this something we need to clarify a bit more? Like, create a new folder and copy the files to it?

Reply:
Actually that is a good question. I would actually prefer to add all the data to the zip file, but that makes version controlling it so much harder.